docs/modules/ROOT/pages/how-tos/crossplane_service_broker/bearer_token_authentication.adoc

```diff
@@ -0,0 +1,182 @@
+== Prepare HTTP _Bearer Token_ Authentication
+
+This how-to will prepare you for the final step, which will be xref:app-catalog:ROOT:how-tos/crossplane_service_broker/connect_service_catalog_to_service_broker.adoc[to connect the _Service Catalog_ to the _Service Broker_].
```

Comment:
I would leave "final step" away, this page lives on its own.

Reply:
I've rephrased this now.